packages/api/src/command/medical/organization/update-organization.ts

```diff
@@ -29,9 +33,41 @@ export const updateOrganization = async (
   const fhirOrg = toFHIR(updatedOrg);
   await upsertOrgToFHIRServer(updatedOrg.cxId, fhirOrg);
 
-  if (org.type !== "healthcare_it_vendor") {
+  if (org.type === "healthcare_provider") {
+    // TODO Move to external/hie
```

Comment:
let's tie any TODOs in the code to a specific issue number

Reply:
Done.